### ISSUE TITLE
Resolve script warnings

### ISSUE THREAD

**Issue by ChristopherIMeyers** [ChristopherIMeyers/ATZ-ReplayFlair#9]
```

[user@ip-123-456-789-012 atz-replayflair]$ python readreplay.py
:0: UserWarning: The keyword `bot` in your user_agent may be problematic.
/usr/lib/python2.6/site-packages/praw/decorators.py:88: DeprecationWarning: reddit intends to disable password-based authentication of API clients sometime in the near future. As a result this method will be removed in a future major version of PRAW.

For more information please see:

* Original reddit deprecation notice: https://www.reddit.com/comments/2ujhkr/

* Updated delayed deprecation notice: https://www.reddit.com/comments/37e2mv/

Pass ``disable_warning=True`` to ``login`` to disable this warning.
  warn(msg, DeprecationWarning)
/usr/lib/python2.6/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning


```

**Comment by ChristopherIMeyers** [comment]
Better now 20ce746232999bc8ef7761a265b3712f6da274f9

**Comment by ChristopherIMeyers** [comment]
fixed 82387e744601376ff3e045c42c520886451f7276
